packages/myst-cli/src/process/notebook.ts

```diff
@@ -165,17 +165,20 @@ export async function processNotebookFull(
           value: ensureString(cell.source),
         };
 
-        // Embed outputs in an output block
-        const output: { type: 'output'; id: string; data: IOutput[] } = {
-          type: 'output',
-          id: nanoid(),
-          data: [],
+        const outputsChildren = (cell.outputs as IOutput[]).map((output) => {
+          // Embed outputs in an output block
+          const result: { type: 'output'; id: string; jupyter_data: IOutput } = {
+            type: 'output',
+            id: nanoid(),
```

Comment:
Not certain of the purpose of `id`, but I suspect we actually want to lift this up.